src/components/CartView.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const CartView = ({ cart }) => {
+  let cartTotal = 0;
+
+  const cartProducts = cart?.products.map((product) => {
+    return (
+      <div key={product.id}>
+        <h2>{product.name}</h2>
+      </div>
+    );
+  });
+
+  cart.products.map((product) => {
+    return (cartTotal += parseInt(product.price));
+  });
```

Comment:
thats good, but it is not  a function. Also now you just save total price in variable and do not update the state. You have to have a function that has local variable cartTotal, does that calculation that you do here and then saves it in the state. Saving in the state is important to have DYNAMIC display of your data.